core/connection/manager.go

```diff
@@ -42,15 +41,15 @@ var (
 	ErrAlreadyExists = errors.New("connection already exists")
 	// ErrConnectionCancelled indicates that connection in progress was cancelled by request of api user
 	ErrConnectionCancelled = errors.New("connection was cancelled")
-	// ErrOpenvpnProcessDied indicates that Connect method didn't reach "Connected" phase due to openvpn error
-	ErrOpenvpnProcessDied = errors.New("openvpn process died")
+	// ErrConnectionDied indicates that Connect method didn't reach "Connected" phase due to connection error
+	ErrConnectionDied = errors.New("connection has died")
```

Comment:
Would use ConnectionFailed, since Died would imply that it was alive in the first place.

Reply:
> ConnectionFailed

 better ConnectingFailed